CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
You can label this `0.2.1`, since that will be the version update/fix. Also, could you update the version number in the `nextflow.config`?

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/83/commits/e35451093675decefda2e34c1de5b0952812cf35

Reply:
Thanks. After some discussion, decided to keep "NoData" since that was the existing behaviour.

Reply:
Thanks. Could you also update the version in `nextflow.config`?

Reply:
I saw this was updated and must have missed it before. Thanks.